readinglist/resource.py

```diff
@@ -93,6 +95,12 @@ class RessourceSchema(colander.MappingSchema):
     _id = colander.SchemaNode(colander.String(), missing=colander.drop)
     last_modified = TimeStamp()
 
+    __readonly_fields__ = ('_id', 'last_modified')
```

Comment:
why do you use double-dunder ? is it something from Colander ? if not it's potential trouble when accessing it